packages/api/src/transifexApi.d.ts

```diff
@@ -42,9 +42,9 @@ declare class JsonApiResource {
 }
 
 declare class Collection {
-  async fetch(): void;
-  async getNext(): Collection
-  async getPrevious(): Collection
+  fetch(): Promise<void>;
```

Comment:
Are we removing the `async` because we are returning a `Promise`? Maybe we should keep it?

Reply:
According to TS docs, `async` is not needed in the context of the declaration file. The return value defines the type.